tasks/rh-sign-image-cosign/tests/test-rh-sign-image-cosign-retries.yaml

```diff
@@ -31,12 +31,28 @@ spec:
                     "repository": "quay.io/redhat-pending/test-product----test-image2",
                     "rh-registry-repo": "registry.stage.redhat.io/test-product/test-image2",
                     "registry-access-repo": "registry.access.stage.redhat.com/test-product/test-image2",
-                    "tags": ["retry-tag"]
+                    "tags": ["t1", "t2"]
                   }
                 ]
               }
               EOF
 
+              # create empty cosign verify mock files
+              REF1="quay.io/redhat-pending/test-product----test-image2:t1"
+              touch "$(workspaces.data.path)/$(echo $REF1 | tr '/' '-')"
+              REF2="quay.io/redhat-pending/test-product----test-image2:t2"
+              touch "$(workspaces.data.path)/$(echo $REF2 | tr '/' '-')"
+
+              # first 5 cosign calls should end with success
```

Comment:
This says 5 calls, but the for cycle has only 3 iterations?

Reply:
yeah, I need to fix this.